coaster/sqlalchemy/statemanager.py

```diff
@@ -526,9 +546,13 @@ def __call__(self, *args, **kwargs):
         # Call the transition function
         try:
             result = self.statetransition.func(self.obj, *args, **kwargs)
+        except AbortTransition as e:
+            transition_exception.send(self.obj, transition=self.statetransition, exception=e)
+            return e.args
```

Comment:
This returns a list even if there was only one parameter. That's confusing.

Reply:
Yeah. In case the length is 1, we'll have to just send the first item.

Reply:
Umm, no. What if you have a list of one item? You'll unwrap the list then. Do not assume downstream use case data types.